airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -562,11 +577,38 @@ def extract_xcom(self, pod: V1Pod) -> str:
                 resp,
                 f"if [ -s {PodDefaults.XCOM_MOUNT_PATH}/return.json ]; then cat {PodDefaults.XCOM_MOUNT_PATH}/return.json; else echo __airflow_xcom_result_empty__; fi",  # noqa
             )
-            self._exec_pod_command(resp, "kill -s SIGINT 1")
+            if isinstance(result, str) and result.rstrip() != "__airflow_xcom_result_empty__":
+                try:
+                    json.loads(result)
+                except Exception as ex:
+                    raise ex
```

Comment:
As you raise the exception you receive anyway, you can skip the overhead of the whole try/catch block
```suggestion
                json.loads(result)
```

Reply:
Thx! removed try-except block here